tests/vmcheck/test-livefs.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
+#
+# Copyright (C) 2017 Red Hat Inc.
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place - Suite 330,
+# Boston, MA 02111-1307, USA.
+
+set -e
+
+. ${commondir}/libtest.sh
+. ${commondir}/libvm.sh
+
+set -x
+
+vm_send_test_repo
+
+vm_assert_layered_pkg foo absent
+
+vm_rpmostree install /tmp/vmcheck/repo/packages/x86_64/foo-1.0-1.x86_64.rpm
+vm_assert_status_jq '.deployments|length == 2'
+echo "ok install foo locally"
+
+if vm_cmd rpm -q foo; then
+    assert_not_reached "have foo?"
+fi
+assert_livefs_ok() {
+    vm_rpmostree ex livefs -n > livefs-analysis.txt
+    assert_file_has_content livefs-analysis.txt 'livefs OK (dry run)'
+}
+assert_livefs_ok
+
+vm_assert_status_jq '.deployments|length == 2' '.deployments[0]["live-replaced"]|not' \
+                    '.deployments[1]["live-replaced"]|not'
+vm_rpmostree ex livefs
+vm_cmd rpm -q foo > rpmq.txt
+assert_file_has_content rpmq.txt foo-1.0-1
+vm_assert_status_jq '.deployments|length == 3' '.deployments[0]["live-replaced"]|not' \
+                    '.deployments[1]["live-replaced"]'
+
+echo "ok livefs stage1"
+
+vm_rpmostree install /tmp/vmcheck/repo/packages/x86_64/test-livefs-with-etc-1.0-1.x86_64.rpm
+assert_livefs_ok
+vm_rpmostree ex livefs
+vm_cmd rpm -q foo test-livefs-with-etc > rpmq.txt
+assert_file_has_content rpmq.txt foo-1.0-1 test-livefs-with-etc-1.0-1
```

Comment:
Actually, looking at the test output now, I just realized `assert_file_has_content` doesn't actually handle arguments past `$2`.